tests/unit/test_endpoint_consumer.py

```diff
@@ -436,9 +436,9 @@ def test_consumer_logs_an_error_on_missing_alerting_data(self):
         with self.assertLogs(level="WARNING") as logger:
             _ = self.harness.charm.prometheus_consumer.alerts()
             messages = logger.output
-            self.assertEqual(len(messages), 2)
+            self.assertEqual(len(messages), 1)
             self.assertIn(
-                "Alert rules were found but no usable group or identifier was present", messages[1]
+                "Alert rules were found but no usable group or identifier was present", messages[0]
```

Comment:
Are those assertions needed at all?

Reply:
I mean, it's `test_consumer_logs_an_error_on_missing_alerting_data`, so checking the logs is kinda crucial.